openfisca_us/parameters/gov/states/in/tax/income/exemptions/additional/amount.yaml

```diff
@@ -0,0 +1,10 @@
+description: Additional exemption amount dependent child IN
```

Comment:
Could we make the descriptions full sentences, so they look nice when displayed on web apps/API explorers?